lib/parallel_tests/rspec/runner.rb

```diff
@@ -47,6 +47,15 @@ def line_is_result?(line)
           line =~ /\d+ examples?, \d+ failures?/
         end
 
+        def command_with_seed(cmd, seed)
+          cmd = cmd.sub(/ --order random:\d+/, '')
+          cmd = cmd.sub(/ --order random/, '')
+          cmd = cmd.sub(/ --order rand/, '')
+          cmd = cmd.sub(/ --seed \d+/, '')
+          cmd = cmd.sub(/ --color --tty/, '')
```

Comment:
Yeah, perhaps we could do all this in one substitution with a more complex regex. But performance is really not an issue here. This is only called when a build fails. IMO clarity is more important here.